Dockerfile.caasp-1_0

```diff
@@ -0,0 +1,74 @@
+FROM opensuse:42.2
+RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/SLE-12:/SP2/SLE_12_SP2/ yast_sle_12_sp2
+RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/CASP:/1.0/SLE_12_SP2/ yast_casp_1.0
```

Comment:
Here I'd raise the priority even more, let say `-p 30` so the CaaSP packages override the 42.2 and SP2 packages even if they have higher version - again, to avoid possible problems with version numbers.